twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/data/remote/api/TwitterAPI.kt

```diff
@@ -12,12 +12,11 @@ class TwitterAPI : APIInterface {
 
     private var twitterInstance: Twitter = TwitterFactory(createConfiguration()).instance
 
-    override suspend fun searchAsync(query: String, count: Int): MutableList<Status> =
+    override suspend fun searchAsync(query: String, count: Int, nextQuery: Query?): QueryResult =
```

Comment:
Is this `nextQuery` parameter really necessary? Can't you change the type of the first parameter to _Query_ and use just that one?